target/linux/ath79/image/tiny-tp-link.mk

```diff
@@ -202,6 +202,30 @@ define Device/tplink_tl-wr703n
 endef
 TARGET_DEVICES += tplink_tl-wr703n
 
+define Device/tplink_tl-wr802n-v1
+  $(Device/tplink-4mlzma)
+  SOC := qca9533
+  DEVICE_MODEL := TL-WR802N
+  DEVICE_VARIANT := v1
+  TPLINK_HWID := 0x08020001
+  SUPPORTED_DEVICES += tl-wr802n-v1
+endef
+TARGET_DEVICES += tplink_tl-wr802n-v1
+
+define Device/tplink_tl-wr802n-v2
+  $(Device/tplink-4mlzma)
+  SOC := qca9533
+  DEVICE_MODEL := TL-WR802N
+  DEVICE_VARIANT := v2
+  SUPPORTED_DEVICES += tl-wr802n-v2
```

Comment:
Maybe move this down to have same order of variables as for v1.

Reply:
Done.